scripts/openstack-quickstart-demosetup

```diff
@@ -1121,14 +1121,14 @@ if [ "x$with_tempest" = "xyes" -a -e /etc/tempest/tempest.conf ]; then
     crudini --set /etc/tempest/tempest.conf auth use_dynamic_credentials True
     crudini --set /etc/tempest/tempest.conf auth tempest_roles Member
     crudini --set /etc/tempest/tempest.conf auth admin_domain_name Default
-    crudini --set /etc/tempest/tempest.conf auth admin_tenant_name admin
+    crudini --set /etc/tempest/tempest.conf auth admin_project_name admin
     crudini --set /etc/tempest/tempest.conf auth admin_password $pw
     crudini --set /etc/tempest/tempest.conf auth admin_username admin
     crudini --set /etc/tempest/tempest.conf identity uri $KEYSTONE_PUBLIC_ENDPOINT
     crudini --set /etc/tempest/tempest.conf identity uri_v3 $KEYSTONE_PUBLIC_ENDPOINT_V3
     crudini --set /etc/tempest/tempest.conf identity auth_version v3
     crudini --set /etc/tempest/tempest.conf identity username demo
-    crudini --set /etc/tempest/tempest.conf identity tenant_name demo
+    crudini --set /etc/tempest/tempest.conf identity project_name demo
```

Comment:
Also `username`, `project_name`, `alt_password` and `password` have been [removed](https://opendev.org/openstack/tempest/commit/40847ac3089d02e6b80f370d92762d7f4e15f7b8) from `identity` group.

Reply:
Done